tests/unittest/St30pHandler.cpp

```diff
@@ -0,0 +1,254 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2025 Intel Corporation
+ */
+
+#include "noctx.hpp"
+
+St30pHandler::St30pHandler(st_tests_context* ctx, SessionUserData* sessionUserData,
+                           st30p_tx_ops ops_tx, st30p_rx_ops ops_rx,
+                           uint msPerFramebuffer, bool create, bool start)
+    : Handlers(ctx, sessionUserData, AUDIO_CLOCK_HRTZ),
+      msPerFramebuffer(msPerFramebuffer) {
+  if (ops_tx.name == nullptr || ops_rx.name == nullptr) {
```

Comment:
I thought that this class allows you to create only single TX or only RX session. This does not look like this and always a pair is created.

Reply:
Please look at st30p_redundant_latency

To check out how you can use the Handlers only as Tx or Rx 

The implementation is pretty simple, you just don't create or start session and instead use createTx and startTx session